src/Workspaces/Core/Portable/Recommendations/IRecommendationService.cs

```diff
@@ -16,6 +17,11 @@ internal interface IRecommendationService : ILanguageService
             int position,
             RecommendationServiceOptions options,
             CancellationToken cancellationToken);
```

Comment:
do we need thsi entrypoint anymore?

Reply:
Yes, there are callers other than completion